Makefile

```diff
@@ -185,7 +185,7 @@ github-pages:
 	mkdir -p ~/bin
 	tar -C ~/bin -zxf ~/hugo.tgz hugo
 	hugo version
-	(cd docs && hugo --verbose)
```

Comment:
What does removing this flag do for us?

Reply:
at some stage since hugo 0.101.0 the flag was removed and using it causes hugo to exit with an error code and help message.

Reply:
see https://github.com/fabiolb/fabio/actions/runs/13146105854/job/36684473079

Reply:
Excellent, thank you for the clarification!